src/PostgREST/Config.hs

```diff
@@ -168,6 +169,7 @@ readOptions = do
         <*> (maybe (Right [JSPKey "role"]) parseRoleClaimKey <$> optValue "role-claim-key")
         <*> (maybe ["public"] splitExtraSearchPath <$> optValue "db-extra-search-path")
         <*> ((\x y -> QualifiedIdentifier x <$> y) <$> dbSchema <*> optString "root-spec")
+        <*> (fmap encodeUtf8 <$> C.required "raw-output-media-type" (C.list C.string))
```

Comment:
I made it a required config key to make the behavior as explicit as possible, but I am not sure whats the preferred approach

Reply:
It definitely should not be required. Empty list [] should be default.

Also, I think `text/plain` should remain included since we always have had support for raw output for a `text` type and users have been using `application/octet-stream` for getting this.

Reply:
Ok. Do you mean text/plain and application/octet-stream should be included by default even if the config variable is left empty?

Reply:
@Dansvidania Yes, correct.

Reply:
@steve-chavez ok sounds good, thanks.